src/pages/login/LoginPage.tsx

```diff
@@ -116,4 +116,4 @@ const LoginPage = () => {
   );
 };
 
-export default LoginPage;
+export { LoginPage };
```

Comment:
WTF, it should be `export default`, why you changed it ?

Reply:
@papercoding check inb messenger!

Reply:
@papercoding Fixed!